### PR TITLE
Flux 0.107

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+related:
+  - /influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/histogramquantile/
+---
+
+The `experimental.histogramQuantile()` function approximates a quantile given a histogram that
+approximates the cumulative distribution of the dataset.
+Each input table represents a single histogram.
+The histogram tables must have two columns – a count column (`_value`) and an upper bound column (`le`).
+
+Input tables must include `le` and `_value` columns neither can be part of the group key.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:20]:
```suggestion
Each input table represents a single histogram. Input tables must have two columns—a count column (_value) and an upper bound column (le), and neither column can be part of the group key. 

```

Comment (kelseiv) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:20]: Is there a reason why we're calling it the count column? The `values` column counts what...

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+The count is the number of values that are less than or equal to the upper bound value (`le`).
+Input tables can have any number of records, each representing an entry in the histogram.
+The counts must be monotonically increasing when sorted by upper bound (`le`).
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:23]: The values?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+If any values in the `_value` or `le` columns are `null`, the function returns an error.
+
+Linear interpolation between the two closest bounds is used to compute the quantile.
+If the either of the bounds used in interpolation are infinite,
+then the other finite bound is used and no interpolation is performed.
+
+The output table has the same group key as the input table.
+The function returns the value of the desired quantile from the histogram in the
+`_value` column and drops all columns not part of the group key.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:32]: maybe we should fit lines 31-32 in the first sentence?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+Input tables can have any number of records, each representing an entry in the histogram.
+The counts must be monotonically increasing when sorted by upper bound (`le`).
+If any values in the `_value` or `le` columns are `null`, the function returns an error.
+
+Linear interpolation between the two closest bounds is used to compute the quantile.
+If the either of the bounds used in interpolation are infinite,
+then the other finite bound is used and no interpolation is performed.
+
+The output table has the same group key as the input table.
+The function returns the value of the desired quantile from the histogram in the
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:31]:
```suggestion
The function returns the value of the specified quantile from the histogram in the
```

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/mode.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import "experimental"
+
+from(bucket: "example-bucket")
+  |> filter(fn: (r) =>
+    r._measurement == "example-measurement" and
+    r._field == "example-field"
+  )
+  |> range(start:-12h)
+  |> window(every:10m)
+  |> experimental.mode()
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/mode.md:58]: Maybe for all examples, we show a data input table and results?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+weight: 302
+related:
+  - /influxdb/v2.0/query-data/flux/percentile-quantile/
+  - /influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/quantile/
+  - /{{< latest "influxdb" "v1" >}}/query_language/functions/#percentile, InfluxQL – PERCENTILE()
+---
+
+The `experimental.quantile()` function outputs non-null records with values in
+the `_value` column that fall within the specified quantile or the non-null
+record with the value in the `_value` column that represents the specified quantile.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md:20]:
```suggestion
The `experimental.quantile()` function outputs non-null records with values in
the `_value` column that fall within the specified quantile or represent the specified quantile.
```

Comment (kelseiv) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md:20]: Not clear on the difference between fall within and represent here, if we can clarify

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+import "experimental"
+
+experimental.quantile(
+  q: 0.99,
+  method: "estimate_tdigest",
+  compression: 1000.0
+)
+```
+
+When using the `estimate_tdigest` or `exact_mean` methods, it outputs non-null
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md:36]: it >> the output table/record?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+records with values that fall within the specified quantile.
+
+When using the `exact_selector` method, it outputs the non-null record with the
+value that represents the specified quantile.
+
+## Parameters
+
+### q
+A value between 0 and 1 indicating the desired quantile.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md:45]:
```suggestion
A value between 0 and 1 thats specifies the quantile.
```

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 9]
```diff
@@ -8,6 +8,22 @@ menu:
     name: Flux
 ---
 
+## v0.107.0 [2021-03-09]
+
+### Features
+- Add new [`experimental.kaufmansAMA()` function](/influxdb/v2.0/reference/flux/stdlib/experimental/kaufmansama/) signature.
+- Add new [experimental aggregate function](/influxdb/v2.0/reference/flux/stdlib/experimental/#experimental-functions) signatures.
+- Add `extends` capability to `testcase` block.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/flux.md:16]: Will folks know what `extends` capability is?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/count.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+```js
+import "experimental"
+
+experimental.count()
+```
+
+{{% note %}}
+#### Empty tables
+`experimental.count()` returns `0` for empty tables.
+To keep empty tables in your data, set the following parameters for the following functions:
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/count.md:32]: No, it returns 0 for empty tables. The other functions listed will drop empty tables unless you specifically tell them not to.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/count.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
+---
+title: experimental.count() function
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/count.md:2]: Experimental count doesn't let you specify the column to operate on. It only works on the `_value` column.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+{{% note %}}
+When the quantile falls below the lowest upper bound (`le`),
+interpolation is performed between `minValue` and the lowest upper bound.
+When `minValue` is equal to negative infinity, the lowest upper bound is used.
+{{% /note %}}
+
+## Examples
+
+##### Compute the 90th quantile
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:69]: The plan is to add input and output examples on all functions, but I'm going to do that in another PR based off the flux-restructure branch.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md:20]: Yes, it's how histogram data is structured in Flux. The value column is actually a count of points that fall in a specific bin or range.

**Pull request merged**